ush/forecast_postdet.sh

```diff
@@ -797,57 +786,77 @@ MOM6_out() {
     export COMIN="$ROTDIR/$CDUMP.$PDY/$cyc"
     export COMOUT="$ROTDIR/$CDUMP.$PDY/$cyc"
   fi
-  [[ ! -d $COMOUT ]] && mkdir -m 775 -p $COMOUT
+  [[ ! -d $COMOUT/ocean ]] && mkdir -m 775 -p $COMOUT/ocean
 
-  if [ $inistep = 'cold' ]; then
-    cp $DATA/ufs.cpld.cold.cpl.r.*.nc $COMOUT/
-    cp $DATA/rpointer.cpl $COMOUT/
+  if [ ${inistep:-"restart"} = 'cold' ]; then
+    $NLN $COMOUT/ocean/ufs.cpld.cold.cpl.r.*.nc $DATA/ufs.cpld.cold.cpl.r.*.nc
+    $NLN $COMOUT/ocean/rpointer.cpl $DATA/rpointer.cpl
     status=$?
     exit $status
```

Comment:
Linking a non-existent file will create a dead link.  The status will always be 0, no?

Reply:
I guess it will be non-zero if it fails to create the link. Not sure if that is worth checking though, since if it can't write to $DATA, there are loads of other problems.

Reply:
Fixed